### PR TITLE
Display linkify

### LINKED ISSUE #28: Make links clickable
You should be able to click on links in chat messages.

### PR DESCRIPTION
**Pull request by martinthomson**
Closes #28

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -78,7 +78,15 @@
 var channel = 'bad-attitude';
 
 function clean(text) {
-  return (''+text).trim().replace( /[<>]/g, '' );
+  var i = 0, m;
```
Comment (bifurcation) [inline index.html:81]: Doing this in clean() will apply it to nicks as well as text.  Is that a desired property?  I'm not totally sold.

File: index.html [hunk 2]
```diff
@@ -78,7 +78,15 @@
...
+  text = text.replace(/&/g, '&amp;').replace(/</g, '&gt;');
```
Comment (bifurcation) [inline index.html]: - `<` is `&lt;`, right?  
- Why not replace both sides of the brackets?
- If you're going to fix these things here, you should probably not delete angle brackets above.

**Comment by bifurcation** [comment]
r=me with comments addressed.

File: index.html (hunk 1 above)
Comment (martinthomson) [inline index.html:81]: Yes, this is a desirable property.

File: index.html (hunk 2 above)
Comment (martinthomson) [inline index.html]: Oops, fixing `&lt;` and `&gt;`.

**Pull request merged**